lua/chatgpt/flows/chat/session.lua

```diff
@@ -5,7 +5,7 @@ local scan = require("plenary.scandir")
 local Session = classes.class()
 
 local function parse_date_time(str)
-  local year, month, day, hour, min, sec = string.match(str, "(%d+)-(%d+)-(%d+)_(%d+):(%d+):(%d+)")
+  local year, month, day, hour, min, sec = string.match(str, "(%d+)-(%d+)-(%d+)_(%d+)" .. (package.config:sub(1,1) == "/" and ":(%d+):(%d+)" or "_(%d+)_(%d+)"))
```

Comment:
If underscores are valid for both windows and unix, maybe we could simplify and just always use them instead of conditionally switch?

Reply:
That is simpler and cleaner thanks for the suggestion. I was thinking I should not break compatibility with existing session files. Will make it uniform across.